src/Visualization.jl

```diff
@@ -152,3 +152,14 @@ function Visualization.create_vtk_file(
       celldata=c,nodaldata=n)
   end
 end
+
+function Base.setindex!(
```

Comment:
This new method leads to type-piracy. Non of the types involved in the signature nor the function belongs to this package. We need the think a workaround.

Can you recall me how is the user API to save to vtk in GridapODEs ?

Reply:
Yes, this was also my concern. 

In a driver with a sequence of output files per time step, one typically does something like:
```julia
paraview_collection(filename) do pvd
  for (x,t) in xt
    pvd[t] = createvtk(...)
  end
end
```

The specialization is needed because for the distributed case `createvtk` returns a partitioned array of `PVTKFile`